site/en/configure/memory.md

```diff
@@ -1,22 +1,27 @@
 Project: /_project.yaml
 Book: /_book.yaml
 
-# Running Bazel with Limited RAM
+
+# Optimize Memory
 
 {% include "_buttons.html" %}
 
-This page describes how to use flags to run Bazel with limited RAM.
+This page describes how to limit and reduce the memory Bazel uses.
+
+## Running Bazel with Limited RAM
 
 In certain situations, you may want Bazel to use minimal memory. You can set the
 maximum heap via the startup flag
-[`--host_jvm_args`](/docs/user-manual#host-jvm-args),
+[`--host_jvm_args`](/user-manual#host-jvm-args),
 like `--host_jvm_args=-Xmx2g`.
 
 However, if your builds are big enough, Bazel may throw an `OutOfMemoryError`
 (OOM) when it doesn't have enough memory. You can make Bazel use less memory, at
 the cost of slower incremental builds, by passing the following command flags:
-[`--discard_analysis_cache`](/docs/user-manual#discard-analysis-cache),
-`--nokeep_state_after_build`, and `--notrack_incremental_state`.
+[`--discard_analysis_cache`](/user-manual#discard-analysis-cache),
```

Comment:
This should be `/reference/command-line-reference#flag--discard_analysis_cache`

Reply:
See other comment re user manual vs command line ref.`

On a different note, why is there some command line documentation in the user manual, separate from the command line documentation? Is this for historic reasons (e.g. manual was there before the command line ref was presumably auto-generated based on documentation in the Java code)?

If so, are there plans (or a wish for) the command line docs in the user manual to be removed and any references to be replaced by links to the command line reference? I am guessing the command line ref is the source of truth (is that so?), but it's easy for the user manual to get out of sync with it.